### ISSUE TITLE
Question: What is the different between gmm and bpkg ?

### ISSUE THREAD

**Issue by breneser** [0x00A/gmm#2]
Hello,

I have been looking to see how to use git submodules as means of dependency management for a bash project I am working on and I am aware of the problems and criticisms of doing so. So, I started doing some research and came across this project and in turn bpkg.

I was wondering doesn't bpkg solve the same problem gmm does ? Considering gmm is also using bpkg i am a little bit confused and can not get the motivation behind this project.Is it actually intended for a specific problem.

Is gmm intented to be a language agnostic module manager based on git ? As far as I can tell bpkg is bash package manager. gmm does not focus on bash ?

**Comment by heapwolf** [comment]
> gmm intented to be a language agnostic module manager based on git

Correct. Also, this isn't really finished, it was an idea that i got started but dont really have the time to work on.

**Comment by heapwolf** [comment]
Also, bpkg is awesome, i'd totally use it for bash :)

**Comment by breneser** [comment]
Thank your very much for quick response :) 

bpkg seems well maintained and quite useful. Gave me some more ideas  :)
